lntest/rpc/lnd.go

```diff
@@ -131,7 +131,9 @@ func (h *HarnessRPC) PendingChannels() *lnrpc.PendingChannelsResponse {
 	ctxt, cancel := context.WithTimeout(h.runCtx, DefaultTimeout)
 	defer cancel()
 
-	pendingChansRequest := &lnrpc.PendingChannelsRequest{}
+	pendingChansRequest := &lnrpc.PendingChannelsRequest{
```

Comment:
yeah almost regret for not making `PendingChannels` taking this request as param - I think this works, can come later when it's time to refactor this, say, if we wanna test this `ClosingTxHex` is definitely not there when not asked. Future PRs.